proposals/0448-type-variable-scoping.rst

```diff
@@ -0,0 +1,1167 @@
+  the way type variables are brought into scope to be more like term variables,
+  and less like pattern signatures.
+* `#420`_: Not yet accepted proposal that is an amendment to `#285`_, with a
+  very tiny, technical tweak to a definition.
+
+This proposal supersedes all of the proposals above, including the accepted
+ones. The goal of writing this is to provide a unified framework, so that
+we can design our language with cohesive treatment of scoped type variables,
+instead of simply an agglomeration of features.
+
+Motivation
+----------
+
+With GHC's powerful type-level programming features, we need powerful abilities
+to bring type variables into scope. The proposal defers to the individual proposals
+linked above for motivation for why we generally want these type-level features.
+Individual aspects of this unifying proposal are motivated near where they are
+introduced.
+
+How to read this proposal
+-------------------------
+
+This is a large proposal, with a number of moving parts. The essential reason
+all these moving parts are glued together in just one proposal is so that they
+can be unified by their desire to uphold the Principles_ laid out here. Individual
+components of this proposal can be designed, debated, and implemented separately,
+yet are presented in one document as they are meant to dovetail together nicely.
+
+As currently written, this proposal is not self-contained, in that motivation for
+some individual pieces was not copied from their source proposals. In all cases,
+when this proposal refers to others as inspiration, seeking more information there
+will likely be helpful.
+
+If this proposal is accepted, it may be a good idea to incorporate that motivation,
+etc., right in this proposal here, to make it self-contained. I am happy to do this
+at the direction of the committee.
+
+Principles
+----------
+
+The design here is guided by several high-level principles. These principles are not
+sacro-sanct, such that they admit no exceptions. Instead, we strive to uphold these
+principles, but relax them when doing so is well motivated.
+
+Each principle is meant to be able to stand alone; that is, each one should make sense
+as a language-design desideratum independent of the others. However, the underlying
+theme to all of them is to reduce the expected cognitive burden of readers and writers
+of a language. For example, it is well specified to have a language supporting a class
+of identifiers in scope only when their first letter occurs in an odd-numbered column
+(counting from 0). But such a language would be challenging to program in! The principles
+here are meant to make programming easier. This is subjective and up for debate;
+consideration of this proposal should also include consideration of whether these
+principles are worth upholding.
+
+If `#452`_ is accepted, these principles will be moved to the Principles document described
+in `#452`_.
```

Comment:
Now that #452 has been accepted, you can do so. But you should leave the motivations for the new principles in this proposal.

On the other hand, no need to mention the principles that were merged as part of #452 (and actually come from #378 ). Only the new principles need to be justified.